tests/unit/style/stylesheets.rs

```diff
@@ -292,14 +293,19 @@ impl CSSInvalidErrorReporterTest {
 }
 
 impl ParseErrorReporter for CSSInvalidErrorReporterTest {
-    fn report_error(&self, input: &mut CssParser, position: SourcePosition, message: &str) {
+    fn report_error(&self, input: &mut CssParser, position: SourcePosition, message: &str,
+        servo_url: Option<&ServoUrl>) {
+
+        let url = servo_url.unwrap().clone();
         let location = input.source_location(position);
 
         let errors = self.errors.clone();
         let mut errors = errors.lock().unwrap();
 
         errors.push(
             CSSError{
+
+                url: url,
```

Comment:
added the **url** into the **cserror** struct so that it can be picked up from the **errors**